src/internal/modules/charge-information/forms/charge-element/quantities.js

```diff
@@ -11,17 +11,17 @@ const getFormField = (key, data) => {
   return fields.text(fieldName, {
     controlClass: 'govuk-input govuk-input--width-10',
     label: `${capitalize(key)}${key === 'billable' ? ' (optional)' : ''}`,
+    type: 'number',
     suffix: 'megalitres per year',
-    mapper: 'numberMapper',
     errors: {
-      'any.empty': {
+      'any.required': {
         message: `Enter an ${key} quantity`
```

Comment:
Erm. Really silly thing: Could this end up rendering `Enter **an valid** billable quantity`? If so, could we do something to fix it so that it's not grammatically incorrect?

(I appreciate this is pedantry on my part)

Reply:
I was going to fix this, but then I realised that this might be a bit misleading as the `'any.required'` and `'any.empty'` errors will only ever show up for the authorised quantity because the billable quantity is optional. I'll try to make it a bit clearer

Reply:
Cool. Thank you :)